templates/web/common/includes/display_product_api.tt.html

```diff
@@ -0,0 +1,32 @@
+
+[% IF single_param("jqm") %]
+    [% Lang.app_please_take_pictures.$request_lc %]
+    <button onclick="captureImage();" data-icon="off-camera">[% Lang.app_take_a_picture.$request_lc %]</button>
+    <div id="upload_image_result"></div>
+    <p>[% Lang.app_take_a_picture_note.$request_lc %]</p>
+[% END %]
+
+[% IF request_ref.api_version >= 0.1 %]
```

Comment:
It should be api_version directly now !

```suggestion
[% IF api_version >= 0.1 %]
```